qiita_db/study.py

```diff
@@ -152,6 +152,19 @@ def _lock_non_sandbox(self):
             raise qdb.exceptions.QiitaDBStatusError(
                 "Illegal operation on non-sandbox study!")
 
+    @staticmethod
+    def all_data_types():
+        """Returns list of all the data types available in the system
+
+        Returns
+        -------
+        list of str
+        """
+        with qdb.sql_connection.TRN:
+            sql = "SELECT DISTINCT data_type FROM qiita.data_type"
+            qdb.sql_connection.TRN.add(sql)
+            return qdb.sql_connection.TRN.execute_fetchflatten()
+
     @property
     def status(self):
         r"""The status is inferred by the status of its artifacts"""
```

Comment:
What do you say about updating the docstring (considering that the return type as completely changed)?